system/lib/libc/musl/src/thread/__timedwait.c

```diff
@@ -80,9 +83,13 @@ int __timedwait(volatile int *addr, int val,
 	clockid_t clk, const struct timespec *at, int priv)
 {
 	int cs, r;
-	__pthread_setcancelstate(PTHREAD_CANCEL_DISABLE, &cs);
```

Comment:
Hmm, was this something that I had incorrectly added outside an `#ifdef __EMSCRIPTEN__` block?

Reply:
No now this call has been moved to #else block.